cloudferrylib/os/actions/snapshots.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+from cloudferrylib.base.action import action
+from cloudferrylib.utils.drivers import ssh_file_to_file
+from cloudferrylib.utils import utils
+
+
+LOG = utils.get_log(__name__)
+
+
+class MysqlDump(action.Action):
+
+    def __init__(self, init, cloud):
+        super(MysqlDump, self).__init__(init, cloud)
```

Comment:
Looks like constructor is not really needed here, you're only calling the base class.